dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
+        Custom Designs · Marketing · Branding · And More
+      </span>
+      <div class="whois">
```

Comment:
To better match the designer's mockups:

- Try adding a `<hr>` tag below `<div class="whois">` to get a "horizontal rule" (a horizontal line).
- In `style.css`, select all of your `<hr>` tags by writing `hr { /* style code goes here */ }` and add some CSS to format the line.
- Back in `index.html`, don't forget to add a TODO comment above the `<hr>` tag to remind yourself to come back later and replace this line with the actual static asset our designer forgot to provide us with!